benches/me.rs

```diff
@@ -33,13 +33,13 @@ fn new_plane<T: Pixel>(ra: &mut ChaChaRng, width: usize, height: usize) -> Plane
   p
 }
 
-fn bench_get_sad(b: &mut Bencher, bs: &BlockSize) {
+fn bench_get_sad(b: &mut Bencher, input_type: &(BlockSize, usize)) {
```

Comment:
The `(BlockSize, usize)` could be passed by value, there is no need for reference here.